.zuul.yaml

```diff
@@ -103,3 +132,50 @@
     vars:
       asciidoc_available: true
       doc_available: false
+
+##########################################################
+#                                                        #
+#               Master Zuul Jobs                         #
+#                                                        #
+##########################################################
+- job:
+    name: openstack-meta-content-provider-master
+    override-checkout: main
+    description: |
+      A zuul job building content from OpenDev master release.
+    parent: openstack-meta-content-provider
+    vars:
+      cifmw_operator_build_meta_build: false
+      cifmw_bop_openstack_release: master
+      cifmw_bop_dlrn_baseurl: "https://trunk.rdoproject.org/centos9-master"
+      cifmw_repo_setup_branch: master
+      cifmw_build_containers_registry_namespace: podified-master-centos9
+
+- job:
+    name: watcher-operator-validation-master
```

Comment:
mixed feelings about the naming, because the opendev content is master, but the operato is main, and the job is operator-validation... but it is fine, since the variation is master x antelope, and the operator should follow the PRs branch